ain/utils.py

```diff
@@ -100,7 +101,7 @@ def toBytes(input: Any) -> bytes:
     elif type(input) is int:
         input = bytes.fromhex(padToEven(stripHexPrefix(hex(input))))
     elif input is None:
-        input = bytes([0])
```

Comment:
```python
>>> bytes([0])
b'\x00'
>>> bytes(0)
b''
```

Reply:
Is `toBytes` equivalent and interoperable with `toBuffer` right?
([@ainblockchain/ain-util](https://github.com/ainblockchain/ain-util/blob/master/src/index.ts#L584-L608))
1.  Why don't we add this information on docstring area?
2. Now I understood that `bytes(0)` is right for `input is None` (`v === null || v === undefined`).

Reply:
I fixed the docstring. Did you mean this right? If not, please comment. Thanks!

Reply:
+1 @kriii